frontend/package.json

```diff
@@ -73,6 +73,7 @@
     "react-dnd": "^2.6.0",
     "react-dnd-html5-backend": "^2.6.0",
     "react-dom": "16.x",
+    "react-dropzone": "4.x",
```

Comment:
Are you concerned about using a module that is [not actively maintained](https://github.com/react-dropzone/react-dropzone/issues/479)?

Reply:
The issue is a bit misleading, since the PRs are being reviewed and merged. When you check the conversation, its about drafting additional reviewers, since the main contributer is there alone.

Reply:
It looks like the existing react-dnd library we have can handle file drops. Can you investigate?

Would be good to avoid having two libraries that can do the same thing.

Reply:
I'm looking at

http://react-dnd.github.io/react-dnd/docs-drop-target.html#handling-files-and-urls

> When using the HTML5 backend, you can handle the file drops just by registering a drop target for HTML5Backend.NativeTypes.FILE or HTML5Backend.NativeTypes.URL built-in types.